docs/reference/actions.md

```diff
@@ -61,6 +66,7 @@ Example usage:
     distro: Ubuntu-20.04
     working-dir: /tmp/github/
     submodules: true
+    token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Since you mention the token permissions, I think it'd be a good idea to demonstrate the token permissions being restricted at the job level.

Reply:
Both suggestions would work as well.